### PR TITLE
rtmp-services: Add Twitter / Periscope as a service

### PR DESCRIPTION
**Pull request by canbal**

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -904,6 +904,53 @@
...
+                    "url": "rtmp://jp.pscp.tv:80/x"
+                },
+                {
+                    "name": "Asia/Pacific: Singapore",
+                    "url": "rtmp://sg.pscp.tv:80/x"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "max video bitrate": 800,
```
Comment (johnboiles) [inline plugins/rtmp-services/data/services.json]: These set sane defaults right? You can still override with higher manually if you want, correct?

Comment (DreadWingKnight) [inline plugins/rtmp-services/data/services.json]: those values should be set to match up with what the streaming service have in their documentation.

**Comment by jp9000** [comment]
800 bitrate is almost nothing for desktop streaming.  If possible please show a citation for this.

**Comment by LtRoyalShrimp** [comment]
Here's the Periscope broad info, which recommends 800. https://help.pscp.tv/customer/en/portal/articles/2600293-what-is-periscope-producer

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (canbal) [inline plugins/rtmp-services/data/services.json]: In my tests, these actually didn't really have any effect, I had to manually set the bitrate and keyint regardless.

These values are our advertised recommended settings.

**Comment by dodgepong** [comment]
It's worth noting that Periscope is primarily a mobile streaming platform, so the bitrate is lower than normal desktop streaming services to accommodate mobile data speeds.

**Comment by canbal** [comment]
Here's the documentation on recommended settings: https://www.periscope.tv/help/external-encoders

Screenshot as well:
image

**Comment by johnboiles** [comment]
For Twitter/Periscope users without transcoding enabled, the broadcasting bitrate is what viewers will see. As @dodgepong mentions, we're primarily a mobile streaming platform; going higher means that fewer people will be able to watch the stream.

File: plugins/rtmp-services/data/services.json (hunk 1 above)
Comment (RytoEX) [inline plugins/rtmp-services/data/services.json]: @canbal 
The settings here would only override user specified settings if you had "Enforce streaming service encoder settings" checked (for Output Mode Simple: Settings > Output > Streaming > Enable Advanced Encoder Settings; for Output Mode Advanced:  Settings > Output > Streaming).

**Comment by RytoEX** [comment]
Changes to rtmp-services services.json requires an update to package.json as well.  See PR #967 for an example of this.  Though, the version number used here will depend on which PR would be merged first.

**Comment by canbal** [comment]
Thanks @RytoEX for pointing that out. I updated the PR hoping that I can get in first, if not I'll update again.

**Pull request merged**